src/app/initTchap.ts

```diff
@@ -0,0 +1,53 @@
+import defaultDispatcher from "matrix-react-sdk/src/dispatcher/dispatcher";
+import { ActionPayload } from "matrix-react-sdk/src/dispatcher/payloads";
+import PlatformPeg from "matrix-react-sdk/src/PlatformPeg";
+
+import TchapClientUtils from "../util/TchapClientUtils";
+import TchapUserSettings from "../util/TchapUserSettings";
+
+/**
+ * Force refresh after the client has started
+ */
+export function attachRefreshHandler() {
```

Comment:
This function attaches the handler, but also does something, and then detaches the handler. So :
```suggestion
export function queueClearCacheAndReload() {
```

Reply:
nice